cfme/scripting/bz.py

```diff
@@ -26,16 +26,23 @@
 import click
 import pytest
 import yaml
+from bugzilla_data import BugzillaData
 
+from cfme.utils import conf
 from cfme.utils.blockers import BZ
 from cfme.utils.log import logger
+from cfme.utils.path import data_path
+from cfme.utils.version import current_version
 
 STATUS = {
     "open_bzs": {"val": "true", "text": "are open", "coverage_text": " open"},
     "closed_bzs": {"val": "false", "text": "are closed", "coverage_text": " closed"},
     "all_bzs": {"val": None, "text": "have coverage", "coverage_text": ""}
 }
 
+QUERY_PATH = data_path.join("/bugzilla-queries/")
+BZ_URL = "bugzilla.redhat.com"
```

Comment:
I would take this url from `data_yaml` than hard-codding it.

Reply:
good idea -- let me do that. I am confused by the travis failures as well, that test runs fine locally.